jsonld.js/src/test/scala/org/w3/banana/jsonldjs/JsonLdJsTest.scala

```diff
@@ -1,13 +1,13 @@
 package org.w3.banana
 package jsonldjs
 
-import org.scalatest.{Matchers, WordSpec}
+import org.scalatest.{ Matchers, WordSpec }
```

Comment:
I was looking for the style guidelines to check if we really had decided to put spaces after the squigly brackets... I would not otherwise know to decide which was better.